examples/tensorflow/mnist.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: jobset.x-k8s.io/v1alpha2
+kind: JobSet
+metadata:
+  name: tensorflow
+spec:
+  replicatedJobs:
+  - name: tensorflow
+    template:
+      spec:
+        parallelism: 2
+        completions: 2
+        backoffLimit: 5
+        template:
+          spec:
+            containers:
+            - name: tensorflow
+              image: kubeflow/tf-mnist-with-summaries:latest
```

Comment:
I would suggest using `docker.io/kubeflowkatib/tf-mnist-with-summaries:latest` instead of this image because this image is no longer maintained.

Reply:
updated.

Reply:
It seems that there is some debate on using this image or just using the example that is updated.  

TBH I was pretty happy with this example and it looks like it clearly works.

WDYT @tenzen-y ?

Reply:
In the JobSet example, I'm ok with your approach :)